projects/laji/src/app/shared-modules/technical-news/technical-news-dumb/technical-news-dumb.component.ts

```diff
@@ -9,14 +9,24 @@ import { News } from '../../../shared/model/News';
 })
 export class TechnicalNewsDumbComponent {
   @Input() set news(news: News[] | null) {
-    this.technicalNews = news?.filter(newsItem => {
-      const days = 1;
-      const isNew = Date.now() - parseInt(newsItem.posted, 10) < (days * 86400000); // number of milliseconds in a day
-      const isTechnical = newsItem.tag === 'technical';
-      return isTechnical && isNew;
-    });
+    if (news) {
+      this.technicalNews = news?.filter(newsItem => {
```

Comment:
This could be changed to `this.techicalNews = (news || []).filter(...`, then the if clause can be removed and the type can be array (`technicalNews: News[] = [];`)